tests/twitter.test.ts

```diff
@@ -8,7 +8,7 @@ afterEach(() => server.resetHandlers());
 afterAll(() => server.close());
 
 describe("Twitter", () => {
-  test("Throw on missing env", async () => {
+  test("Throw on missing env variable", async () => {
```

Comment:
Maybe remove this test if it's designed to fail.

Reply:
We are supposed to test for failure as it is expected to have a log, otherwise that is code we do not cover

Reply:
Can we suppress the error in the logs? That's what confused me

Reply:
I can have a look if I can pipe the stderr to silence it then

Reply:
@pavlovcik now during that test no error is logged to the console